internal/oidc/login/loginhtml/loginhtml.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 the Pinniped contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package loginhtml defines HTML templates used by the Supervisor.
+//nolint: gochecknoglobals // This package uses globals to ensure that all parsing and minifying happens at init.
+package loginhtml
+
+import (
+	_ "embed" // Needed to trigger //go:embed directives below.
+	"html/template"
+	"strings"
+
+	"github.com/tdewolff/minify/v2/minify"
+
+	"go.pinniped.dev/internal/oidc/provider/csp"
+)
+
+var (
+	//go:embed login_form.css
+	rawCSS      string
+	minifiedCSS = mustMinify(minify.CSS(rawCSS))
```

Comment:
Changed the name of `mustMinify` in 39fd9ba2701477db1f8b1032e1fb6fe8c8f87a15 to make it more clear what it does. This code mirrors the approach used previously in `formposthtml.go` where a similar helper is used for both CSS and JS minification. Seemed reasonable to leave the code purposely similar since the two packages are basically implementing the same things for different UI pages.